configs/_base_/schedules/imagenet_bs4096.py

```diff
@@ -0,0 +1,13 @@
+# optimizer
+optimizer = dict(
+    type='AdamW', lr=0.0015,
+    weight_decay=0.3)  # 0.003 / 4096 * (32bs * 64gpus) = 0.0015
```

Comment:
The lr configuration is for batch_size = 2048. So (i) we'd better rename the filename as imagenet_bs2048.py; (ii) it's better to explain more about the equation 0.003 / 4096 * (32bs * 64gpus) = 0.0015; maybe add some text to explain this.